plugins/outputs/prometheus_client/README.md

```diff
@@ -10,6 +10,16 @@ This plugin starts a [Prometheus](https://prometheus.io/) Client, it exposes all
   # Address to listen on
   listen = ":9273"
 
+  # Use TLS
+  tls = true
+  tls_crt = "/etc/ssl/telegraf.crt"
```

Comment:
Call this `tls_cert`

Reply:
Done in https://github.com/influxdata/telegraf/pull/3719/commits/de9d7faf312bd7d5c425764fbbcf5228a6590b93